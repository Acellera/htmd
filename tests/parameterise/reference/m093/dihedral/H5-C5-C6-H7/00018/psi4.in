set {
	reference rhf
	basis 6-31G*
}


set_num_threads( 1 )
memory 2 gb

molecule MOL {
0 1

O	 -2.039815	 -0.478655	 -4.482484
C	 -2.611156	 -1.317762	 -3.854866
C	 -2.339648	 -1.500676	 -2.371919
C	 -3.650617	 -2.214202	 -4.489918
C	 -0.938215	 -2.088216	 -2.139699
H	 -3.087207	 -2.183153	 -1.982453
H	 -2.437797	 -0.555327	 -1.845889
C	 0.149006	 -1.302371	 -2.876593
H	 -0.747190	 -2.095437	 -1.072534
H	 -0.916495	 -3.125606	 -2.469999
C	 0.075323	 -1.399736	 -4.404711
H	 1.126728	 -1.643891	 -2.541704
H	 0.078091	 -0.254534	 -2.592241
C	 -1.053427	 -2.288511	 -4.930067
H	 1.023541	 -1.787350	 -4.768483
H	 -0.024340	 -0.400645	 -4.824201
H	 -0.984148	 -3.291584	 -4.518035
H	 -1.004790	 -2.372263	 -6.011279
H	 -2.032469	 -1.894716	 -4.678683
H	 -3.293665	 -3.240761	 -4.495005
H	 -3.843411	 -1.896191	 -5.505046
H	 -4.573197	 -2.199013	 -3.916838

	symmetry c1
}
set optking {
	frozen_dihedral = ("
		6 3 2 1,
		6 3 5 8,
		3 5 8 12
	")
}

ee = energy('scf')
oeprop('DIPOLE', 'QUADRUPOLE', 'MULLIKEN_CHARGES')
f=open( 'psi4out.xyz', 'w' )
f.write( "22  " )
f.write( str(ee) + "\n" )
f.write( MOL.save_string_xyz() )
f.close()
