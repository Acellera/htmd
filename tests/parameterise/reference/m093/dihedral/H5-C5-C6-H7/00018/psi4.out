    -----------------------------------------------------------------------
          Psi4: An Open-Source Ab Initio Electronic Structure Package
                              Psi4 (no tag) Driver

                          Git: Rev {detached?}  

    J. M. Turney, A. C. Simmonett, R. M. Parrish, E. G. Hohenstein,
    F. A. Evangelista, J. T. Fermann, B. J. Mintz, L. A. Burns, J. J. Wilke,
    M. L. Abrams, N. J. Russ, M. L. Leininger, C. L. Janssen, E. T. Seidl,
    W. D. Allen, H. F. Schaefer, R. A. King, E. F. Valeev, C. D. Sherrill,
    and T. D. Crawford, WIREs Comput. Mol. Sci. 2, 556-565 (2012)
    (doi: 10.1002/wcms.93)

                         Additional Contributions by
    A. E. DePrince, M. Saitow, U. Bozkaya, A. Yu. Sokolov
    -----------------------------------------------------------------------


    Psi4 started on: Tue Jul  5 17:22:33 2016

    Process ID:  21035
    PSI4DATADIR: /export131/home/mjharvey/miniconda3/share
    Memory level set to 256.000 MB

  ==> Input File <==

--------------------------------------------------------------------------
set {
	reference rhf
	basis 6-31G*
}


set_num_threads( 1 )
memory 2 gb

molecule MOL {
0 1

O	 -2.039815	 -0.478655	 -4.482484
C	 -2.611156	 -1.317762	 -3.854866
C	 -2.339648	 -1.500676	 -2.371919
C	 -3.650617	 -2.214202	 -4.489918
C	 -0.938215	 -2.088216	 -2.139699
H	 -3.087207	 -2.183153	 -1.982453
H	 -2.437797	 -0.555327	 -1.845889
C	 0.149006	 -1.302371	 -2.876593
H	 -0.747190	 -2.095437	 -1.072534
H	 -0.916495	 -3.125606	 -2.469999
C	 0.075323	 -1.399736	 -4.404711
H	 1.126728	 -1.643891	 -2.541704
H	 0.078091	 -0.254534	 -2.592241
C	 -1.053427	 -2.288511	 -4.930067
H	 1.023541	 -1.787350	 -4.768483
H	 -0.024340	 -0.400645	 -4.824201
H	 -0.984148	 -3.291584	 -4.518035
H	 -1.004790	 -2.372263	 -6.011279
H	 -2.032469	 -1.894716	 -4.678683
H	 -3.293665	 -3.240761	 -4.495005
H	 -3.843411	 -1.896191	 -5.505046
H	 -4.573197	 -2.199013	 -3.916838

	symmetry c1
}
set optking {
	frozen_dihedral = ("
		6 3 2 1,
		6 3 5 8,
		3 5 8 12
	")
}

ee = energy('scf')
oeprop('DIPOLE', 'QUADRUPOLE', 'MULLIKEN_CHARGES')
f=open( 'psi4out.xyz', 'w' )
f.write( "22  " )
f.write( str(ee) + "\n" )
f.write( MOL.save_string_xyz() )
f.close()
--------------------------------------------------------------------------

  Memory set to   2.000 GiB by Python script.

*** tstart() called on cx1-3-4-4.cx1.hpc.ic.ac.uk
*** at Tue Jul  5 17:22:33 2016


         ---------------------------------------------------------
                                   SCF
            by Justin Turney, Rob Parrish, and Andy Simmonett
                              RHF Reference
                        1 Threads,   2000 MiB Core
         ---------------------------------------------------------

  ==> Geometry <==

    Molecular point group: c1
    Full point group: C1

    Geometry (in Angstrom), charge = 0, multiplicity = 1:

       Center              X                  Y                   Z               Mass       
    ------------   -----------------  -----------------  -----------------  -----------------
           O         -0.480455077028     1.100117704456    -0.765421469158    15.994914619560
           C         -1.051796077028     0.261010704456    -0.137803469158    12.000000000000
           C         -0.780288077028     0.078096704456     1.345143530842    12.000000000000
           C         -2.091257077028    -0.635429295544    -0.772855469158    12.000000000000
           C          0.621144922972    -0.509443295544     1.577363530842    12.000000000000
           H         -1.527847077028    -0.604380295544     1.734609530842     1.007825032070
           H         -0.878437077028     1.023445704456     1.871173530842     1.007825032070
           C          1.708365922972     0.276401704456     0.840469530842    12.000000000000
           H          0.812169922972    -0.516664295544     2.644528530842     1.007825032070
           H          0.642864922972    -1.546833295544     1.247063530842     1.007825032070
           C          1.634682922972     0.179036704456    -0.687648469158    12.000000000000
           H          2.686087922972    -0.065118295544     1.175358530842     1.007825032070
           H          1.637450922972     1.324238704456     1.124821530842     1.007825032070
           C          0.505932922972    -0.709738295544    -1.213004469158    12.000000000000
           H          2.582900922972    -0.208577295544    -1.051420469158     1.007825032070
           H          1.535019922972     1.178127704456    -1.107138469158     1.007825032070
           H          0.575211922972    -1.712811295544    -0.800972469158     1.007825032070
           H          0.554569922972    -0.793490295544    -2.294216469158     1.007825032070
           H         -0.473109077028    -0.315943295544    -0.961620469158     1.007825032070
           H         -1.734305077028    -1.661988295544    -0.777942469158     1.007825032070
           H         -2.284051077028    -0.317418295544    -1.787983469158     1.007825032070
           H         -3.013837077028    -0.620240295544    -0.199775469158     1.007825032070

  Running in c1 symmetry.

  Rotational constants: A =      0.09887  B =      0.06677  C =      0.05184 [cm^-1]
  Rotational constants: A =   2963.97341  B =   2001.72556  C =   1554.23804 [MHz]
  Nuclear repulsion =  430.540507698479644

  Charge       = 0
  Multiplicity = 1
  Electrons    = 64
  Nalpha       = 32
  Nbeta        = 32

  ==> Algorithm <==

  SCF Algorithm Type is DF.
  DIIS enabled.
  MOM disabled.
  Fractional occupation disabled.
  Guess Type is CORE.
  Energy threshold   = 1.00e-06
  Density threshold  = 1.00e-06
  Integral threshold = 0.00e+00

  ==> Primary Basis <==

  Basis Set: 6-31G*
    Number of shells: 76
    Number of basis function: 148
    Number of Cartesian functions: 148
    Spherical Harmonics?: false
    Max angular momentum: 2

  ==> Pre-Iterations <==

   -------------------------------------------------------
    Irrep   Nso     Nmo     Nalpha   Nbeta   Ndocc  Nsocc
   -------------------------------------------------------
     A        148     148       0       0       0       0
   -------------------------------------------------------
    Total     148     148      32      32      32       0
   -------------------------------------------------------

 OEINTS: Overlap, kinetic, potential, dipole, and quadrupole integrals
         stored in file 35.

  ==> Integral Setup <==

  ==> DFJK: Density-Fitted J/K Matrices <==

    J tasked:                  Yes
    K tasked:                  Yes
    wK tasked:                  No
    OpenMP threads:              1
    Integrals threads:           1
    Memory (MB):              1430
    Algorithm:                Core
    Integral Cache:           NONE
    Schwarz Cutoff:          1E-12
    Fitting Condition:       1E-12

   => Auxiliary Basis Set <=

  Basis Set: 
    Number of shells: 318
    Number of basis function: 998
    Number of Cartesian functions: 998
    Spherical Harmonics?: false
    Max angular momentum: 3

  Minimum eigenvalue in the overlap matrix is 1.8285462211E-03.
  Using Symmetric Orthogonalization.
  SCF Guess: Core (One-Electron) Hamiltonian.

  ==> Iterations <==

                           Total Energy        Delta E     RMS |[F,P]|

   @DF-RHF iter   1:  -261.90964341056889   -2.61910e+02   6.87745e-02 
   @DF-RHF iter   2:  -264.98174296108385   -3.07210e+00   4.84216e-02 DIIS
   @DF-RHF iter   3:  -313.70105197850347   -4.87193e+01   4.53250e-02 DIIS
   @DF-RHF iter   4:  -343.92377108977240   -3.02227e+01   1.61930e-02 DIIS
   @DF-RHF iter   5:  -346.76131280306601   -2.83754e+00   9.48750e-03 DIIS
   @DF-RHF iter   6:  -347.74610736386836   -9.84795e-01   2.41964e-03 DIIS
   @DF-RHF iter   7:  -347.83589695964014   -8.97896e-02   5.30154e-04 DIIS
   @DF-RHF iter   8:  -347.84098028103261   -5.08332e-03   1.84954e-04 DIIS
   @DF-RHF iter   9:  -347.84175789477121   -7.77614e-04   6.08035e-05 DIIS
   @DF-RHF iter  10:  -347.84188268400447   -1.24789e-04   3.11060e-05 DIIS
   @DF-RHF iter  11:  -347.84191003896154   -2.73550e-05   1.23417e-05 DIIS
   @DF-RHF iter  12:  -347.84191743356513   -7.39460e-06   6.45946e-06 DIIS
   @DF-RHF iter  13:  -347.84191959560025   -2.16204e-06   2.56281e-06 DIIS
   @DF-RHF iter  14:  -347.84192005607366   -4.60473e-07   9.71653e-07 DIIS

  ==> Post-Iterations <==

    Orbital Energies (a.u.)
    -----------------------

    Doubly Occupied:                                                      

       1A    -20.499451     2A    -11.343481     3A    -11.235084  
       4A    -11.234038     5A    -11.228206     6A    -11.208266  
       7A    -11.195824     8A    -11.182725     9A     -1.428860  
      10A     -1.099025    11A     -1.047314    12A     -1.024939  
      13A     -0.930966    14A     -0.874175    15A     -0.800136  
      16A     -0.727804    17A     -0.695376    18A     -0.672346  
      19A     -0.645391    20A     -0.616080    21A     -0.593862  
      22A     -0.571019    23A     -0.562270    24A     -0.555185  
      25A     -0.528044    26A     -0.513353    27A     -0.506772  
      28A     -0.459645    29A     -0.445864    30A     -0.435544  
      31A     -0.389288    32A     -0.356201  

    Virtual:                                                              

      33A      0.141127    34A      0.219483    35A      0.262245  
      36A      0.272482    37A      0.277456    38A      0.293234  
      39A      0.294487    40A      0.305651    41A      0.324951  
      42A      0.331276    43A      0.337948    44A      0.354720  
      45A      0.368859    46A      0.372602    47A      0.399053  
      48A      0.408362    49A      0.452702    50A      0.459229  
      51A      0.466415    52A      0.499585    53A      0.525010  
      54A      0.605449    55A      0.695274    56A      0.733771  
      57A      0.755275    58A      0.767246    59A      0.785714  
      60A      0.800937    61A      0.811300    62A      0.853623  
      63A      0.860480    64A      0.901981    65A      0.917655  
      66A      0.932488    67A      0.947677    68A      0.978126  
      69A      1.001679    70A      1.012546    71A      1.044907  
      72A      1.077865    73A      1.100839    74A      1.126339  
      75A      1.138464    76A      1.149882    77A      1.151473  
      78A      1.181802    79A      1.187436    80A      1.188960  
      81A      1.197557    82A      1.208138    83A      1.215147  
      84A      1.227245    85A      1.237737    86A      1.256791  
      87A      1.276626    88A      1.295696    89A      1.313568  
      90A      1.394350    91A      1.440790    92A      1.457868  
      93A      1.520158    94A      1.548993    95A      1.623529  
      96A      1.638133    97A      1.696489    98A      1.723901  
      99A      1.806478   100A      1.856564   101A      1.887191  
     102A      1.959639   103A      2.011896   104A      2.063387  
     105A      2.074425   106A      2.084701   107A      2.135698  
     108A      2.178502   109A      2.191783   110A      2.220981  
     111A      2.245905   112A      2.267320   113A      2.278732  
     114A      2.334079   115A      2.378992   116A      2.387952  
     117A      2.400076   118A      2.434927   119A      2.502752  
     120A      2.534437   121A      2.541251   122A      2.607398  
     123A      2.628164   124A      2.654221   125A      2.683541  
     126A      2.717788   127A      2.724258   128A      2.745101  
     129A      2.808852   130A      2.844815   131A      2.889222  
     132A      2.920906   133A      2.927917   134A      2.991207  
     135A      3.041050   136A      3.108119   137A      3.141472  
     138A      3.149997   139A      3.395085   140A      3.457744  
     141A      4.574559   142A      4.647238   143A      4.700185  
     144A      4.729111   145A      4.839647   146A      4.925267  
     147A      5.090662   148A      5.106486  

    Final Occupation by Irrep:
              A 
    DOCC [    32 ]

  Energy converged.

  @DF-RHF Final Energy:  -347.84192005607366

   => Energetics <=

    Nuclear Repulsion Energy =            430.5405076984796438
    One-Electron Energy =               -1325.5517789911696127
    Two-Electron Energy =                 547.1693512366163077
    DFT Exchange-Correlation Energy =       0.0000000000000000
    Empirical Dispersion Energy =           0.0000000000000000
    PCM Polarization Energy =               0.0000000000000000
    EFP Energy =                            0.0000000000000000
    Total Energy =                       -347.8419200560737181

    Alert: EFP and PCM quantities not currently incorporated into SCF psivars.

Properties will be evaluated at   0.000000,   0.000000,   0.000000 Bohr
  ==> Properties <==


Properties computed using the SCF density matrix

  Nuclear Dipole Moment: (a.u.)
     X:     1.0427      Y:    -4.5299      Z:     0.7618

  Electronic Dipole Moment: (a.u.)
     X:    -1.5752      Y:     3.7534      Z:    -0.1413

  Dipole Moment: (a.u.)
     X:    -0.5325      Y:    -0.7765      Z:     0.6204     Total:     1.1276

  Dipole Moment: (Debye)
     X:    -1.3535      Y:    -1.9737      Z:     1.5769     Total:     2.8660


  Saving occupied orbitals to File 180.

*** tstop() called on cx1-3-4-4.cx1.hpc.ic.ac.uk at Tue Jul  5 17:22:53 2016
Module time:
	user time   =      19.07 seconds =       0.32 minutes
	system time =       0.20 seconds =       0.00 minutes
	total time  =         20 seconds =       0.33 minutes
Total time:
	user time   =      19.07 seconds =       0.32 minutes
	system time =       0.20 seconds =       0.00 minutes
	total time  =         20 seconds =       0.33 minutes


Properties will be evaluated at   0.000000,   0.000000,   0.000000 Bohr

Properties computed using the  density matrix

  Nuclear Dipole Moment: (a.u.)
     X:     1.0427      Y:    -4.5299      Z:     0.7618

  Electronic Dipole Moment: (a.u.)
     X:    -1.5752      Y:     3.7534      Z:    -0.1413

  Dipole Moment: (a.u.)
     X:    -0.5325      Y:    -0.7765      Z:     0.6204     Total:     1.1276

  Dipole Moment: (Debye)
     X:    -1.3535      Y:    -1.9737      Z:     1.5769     Total:     2.8660

  Quadrupole Moment: (Debye Ang)
    XX:   -49.1062     YY:   -54.9886     ZZ:   -51.3472
    XY:     1.7113     XZ:    -0.0264     YZ:     2.1713

  Traceless Quadrupole Moment: (Debye Ang)
    XX:     2.7078     YY:    -3.1746     ZZ:     0.4668
    XY:     1.7113     XZ:    -0.0264     YZ:     2.1713

  Mulliken Charges: (a.u.)
   Center  Symbol    Alpha    Beta     Spin     Total
       1     O     4.29535  4.29535  0.00000 -0.59070
       2     C     2.78925  2.78925  0.00000  0.42151
       3     C     3.18366  3.18366  0.00000 -0.36732
       4     C     3.28786  3.28786  0.00000 -0.57573
       5     C     3.17067  3.17067  0.00000 -0.34134
       6     H     0.41509  0.41509  0.00000  0.16982
       7     H     0.39524  0.39524  0.00000  0.20953
       8     C     3.15302  3.15302  0.00000 -0.30605
       9     H     0.41210  0.41210  0.00000  0.17580
      10     H     0.41530  0.41530  0.00000  0.16939
      11     C     3.15610  3.15610  0.00000 -0.31220
      12     H     0.42011  0.42011  0.00000  0.15978
      13     H     0.41560  0.41560  0.00000  0.16880
      14     C     3.22659  3.22659  0.00000 -0.45318
      15     H     0.42336  0.42336  0.00000  0.15328
      16     H     0.39362  0.39362  0.00000  0.21277
      17     H     0.44102  0.44102  0.00000  0.11796
      18     H     0.41116  0.41116  0.00000  0.17767
      19     H     0.39940  0.39940  0.00000  0.20120
      20     H     0.39909  0.39909  0.00000  0.20182
      21     H     0.39219  0.39219  0.00000  0.21562
      22     H     0.40422  0.40422  0.00000  0.19156

   Total alpha = 32.00000, Total beta = 32.00000, Total charge =  0.00000


*** PSI4 exiting successfully. Buy a developer a beer!
